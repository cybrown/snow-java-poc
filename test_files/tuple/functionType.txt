type MyDelegate fun (num: int): string
