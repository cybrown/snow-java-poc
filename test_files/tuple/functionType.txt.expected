<SnowType>
    <Identifier>MyDelegate</Identifier>
    <TypeFunction>
        <TypeFunctionParameterList>
            <TypeFunctionParameter>
                <Identifier>num</Identifier>
                <Identifier>int</Identifier>
            </TypeFunctionParameter>
        </TypeFunctionParameterList>
        <Identifier>string</Identifier>
    </TypeFunction>
</SnowType>
